### ISSUE TITLE
Windows 8 installation

### ISSUE THREAD

**Issue by bdbais** [mathiasertl/fabric#10]
I'm trying to install Fabric3, install miss some dependencies on windows machine and when I try to install them I found error:

C:\projects\testfab>C:\Users\user\.virtualenvs\test\Scripts\fab.exe fabfile.
py
Traceback (most recent call last):
  File "c:\users\user\.virtualenvs\test\lib\site-packages\fabric\network.py",
 line 21, in <module>
    import paramiko as ssh
  File "c:\users\user\.virtualenvs\test\lib\site-packages\paramiko\__init__.p
y", line 30, in <module>
    from paramiko.transport import SecurityOptions, Transport
  File "c:\users\user\.virtualenvs\test\lib\site-packages\paramiko\transport.
py", line 50, in <module>
    from paramiko.dsskey import DSSKey
  File "c:\users\user\.virtualenvs\test\lib\site-packages\paramiko\dsskey.py"
, line 26, in <module>
    from Crypto.PublicKey import DSA
  File "c:\users\user\.virtualenvs\test\lib\site-packages\Crypto\PublicKey\DS
A.py", line 89, in <module>
    from Crypto import Random
  File "c:\users\user\.virtualenvs\test\lib\site-packages\Crypto\Random\__ini
t__.py", line 28, in <module>
    from Crypto.Random import OSRNG
  File "c:\users\user\.virtualenvs\test\lib\site-packages\Crypto\Random\OSRNG
\__init__.py", line 34, in <module>
    from Crypto.Random.OSRNG.nt import new
  File "c:\users\user\.virtualenvs\test\lib\site-packages\Crypto\Random\OSRNG
\nt.py", line 28, in <module>
    import winrandom
ImportError: No module named 'winrandom'

There was a problem importing our SSH library (see traceback above).
Please make sure all dependencies are installed and importable.

I tryied to install win

C:\projects\generali>C:\Users\user\.virtualenvs\test\Scripts\pip3.5.exe insta
ll winrandom  --proxy=http://localhost:8124 --no-cache
Collecting winrandom
  Downloading winrandom-1.2.zip
Installing collected packages: winrandom
  Running setup.py install for winrandom ... error
    Complete output from command c:\users\user\.virtualenvs\test\scripts\pyth
on.exe -u -c "import setuptools, tokenize;__file__='C:\\Users\\user\\AppData\\Lo
cal\\Temp\\pip-build-pc86pxzj\\winrandom\\setup.py';exec(compile(getattr(tokeniz
e, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" ins
tall --record C:\Users\user\AppData\Local\Temp\pip-06lbc65m-record\install-recor
d.txt --single-version-externally-managed --compile --install-headers c:\users\u
ser\.virtualenvs\test\include\site\python3.5\winrandom:
    running install
    running build
    running build_ext
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\user\AppData\Local\Temp\pip-build-pc86pxzj\winrandom\setup.
py", line 43, in <module>
        ext_modules=[winrandom1])
      File "c:\users\user\appdata\local\programs\python\python35\Lib\distutils\c
ore.py", line 148, in setup
        dist.run_commands()
      File "c:\users\user\appdata\local\programs\python\python35\Lib\distutils\d
ist.py", line 955, in run_commands
        self.run_command(cmd)
      File "c:\users\user\appdata\local\programs\python\python35\Lib\distutils\d
ist.py", line 974, in run_command
        cmd_obj.run()
      File "c:\users\user\.virtualenvs\test\lib\site-packages\setuptools\comm
and\install.py", line 61, in run
        return orig.install.run(self)
      File "c:\users\user\appdata\local\programs\python\python35\Lib\distutils\c
ommand\install.py", line 539, in run
        self.run_command('build')
      File "c:\users\user\appdata\local\programs\python\python35\Lib\distutils\c
md.py", line 313, in run_command
        self.distribution.run_command(command)
      File "c:\users\user\appdata\local\programs\python\python35\Lib\distutils\d
ist.py", line 974, in run_command
        cmd_obj.run()
      File "c:\users\user\appdata\local\programs\python\python35\Lib\distutils\c
ommand\build.py", line 135, in run
        self.run_command(cmd_name)
      File "c:\users\user\appdata\local\programs\python\python35\Lib\distutils\c
md.py", line 313, in run_command
        self.distribution.run_command(command)
      File "c:\users\user\appdata\local\programs\python\python35\Lib\distutils\d
ist.py", line 974, in run_command
        cmd_obj.run()
      File "c:\users\user\.virtualenvs\test\lib\site-packages\setuptools\comm
and\build_ext.py", line 49, in run
        _build_ext.run(self)
      File "c:\users\user\appdata\local\programs\python\python35\Lib\distutils\c
ommand\build_ext.py", line 307, in run
        force=self.force)
      File "c:\users\user\appdata\local\programs\python\python35\Lib\distutils\c
compiler.py", line 1031, in new_compiler
        return klass(None, dry_run, force)
      File "c:\users\user\appdata\local\programs\python\python35\Lib\distutils\c
ygwinccompiler.py", line 282, in __init__
        CygwinCCompiler.__init__ (self, verbose, dry_run, force)
      File "c:\users\user\appdata\local\programs\python\python35\Lib\distutils\c
ygwinccompiler.py", line 126, in __init__
        if self.ld_version >= "2.10.90":
    TypeError: unorderable types: NoneType() >= str()

    ----------------------------------------
Command "c:\users\user\.virtualenvs\test\scripts\python.exe -u -c "import set
uptools, tokenize;__file__='C:\\Users\\user\\AppData\\Local\\Temp\\pip-build-pc8
6pxzj\\winrandom\\setup.py';exec(compile(getattr(tokenize, 'open', open)(__file_
_).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record C:\Users\u
ser\AppData\Local\Temp\pip-06lbc65m-record\install-record.txt --single-version-e
xternally-managed --compile --install-headers c:\users\user\.virtualenvs\test
\include\site\python3.5\winrandom" failed with error code 1 in C:\Users\user\App
Data\Local\Temp\pip-build-pc86pxzj\winrandom\
```

**Comment by mathiasertl** [comment]
The import error implies that the problem lies with the pycrypto library. With google I found [this issue](https://github.com/dlitz/pycrypto/issues/110). Can you try that and tell me if it works?

If it does, I can update the documentation ;-).

**Comment by Roman-Sky** [comment]
I faced that issue too on Windows 10 and Python 3.5. After some research I found out that this issue was actually fixed and fix is merged in pycrypto (see [pull request #46](https://github.com/dlitz/pycrypto/pull/46). PyPi has old version of pycrypto so I installed fixed version from repo like this:
`pip install git+https://github.com/dlitz/pycrypto.git@10abfc8#egg=pycrypto`
With this version installed I don't get winrandom import error anymore.
